etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml

```diff
@@ -0,0 +1,140 @@
+      # - producer: World Bank
+      #   title: World Development Indicators - World Bank (2023.05.11)
+      #   description: >-
+      #     The World Development Indicators (WDI) is the primary World Bank collection of development indicators, compiled
+      #     from officially-recognized international sources. It presents the most current and accurate global development data
+      #     available, and includes national, regional and global estimates.
+
+
+      #     The World Development Indicators (WDI) is the primary World Bank collection of development indicators, compiled
+      #     from officially-recognized international sources. It presents the most current and accurate global development data
+      #     available, and includes national, regional and global estimates.
+      #   url_main: https://datacatalog.worldbank.org/search/dataset/0037712/World-Development-Indicators
+      #   url_download: http://databank.worldbank.org/data/download/WDI_csv.zip
+      #   date_accessed: '2023-05-29'
+      #   date_published: '2023-05-11'
+      #   license:
+      #     name: Creative Commons Attribution 4.0
+      #     url: https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets
+tables:
+  wdi:
+    variables:
+      sh_sta_stnt_zs:
+        # title: Prevalence of stunting, height for age (% of children under 5)
+        title: Share of children who are stunted
+        unit: '% of children under 5'
+        short_unit: '%'
+        description_short: |-
+          The share of children younger than five years old whose growth is stunted.
+        # description: Prevalence of stunting is the percentage of children under age 5 whose height for age is more than two
+        #   standard deviations below the median for the international reference population ages 0-59 months. For children up
+        #   to two years old height is measured by recumbent length. For older children height is measured by stature while
+        #   standing. The data are based on the WHO's new child growth standards released in 2006.
+        description_key:
+          - Stunting is when a child is significantly shorter than the average for their age.
+          - Stunted growth is a consequence of poor nutrition and/or repeated infection.
+        description_from_producer: |-
+          Prevalence of stunting is the percentage of children under age 5 whose height for age is more than two standard deviations below the median for the international reference population ages 0-59 months. For children up to two years old height is measured by recumbent length. For older children height is measured by stature while standing. The data are based on the WHO's new child growth standards released in 2006.
+
+
+          \[Text from World Bank World Development Indicators [metadata glossary](https://databank.worldbank.org/metadataglossary/world-development-indicators/series/SH.STA.STNT.ZS)\]
```

Comment:
What's the purpose of the `\[`? Is it supposed to render differently on the data page?


![image](https://github.com/owid/etl/assets/18101289/8d7da904-fb8e-407c-a2c5-9f59a5ae0c84)